src/utils.js

```diff
@@ -135,6 +135,7 @@ export default class Utils {
     if (v[0] === 7) return 'Nougat';
     if (v[0] === 8) return 'Oreo';
     if (v[0] === 9) return 'Pie';
+    if (v[0] === 10) return 'Q';
```

Comment:
I'm not sure if Q is the official name for it. It seems Google has stopped calling versions by particular names and switched to numbers of major versions. [Wikipedia says](https://en.wikipedia.org/wiki/Android_10) that it's been a preliminary name for it, but official seems to be just `Android 10`. And the upcoming 11 also seems to be just `Android 11`.
Should it be just `Android 10` then instead of a `Q`?

Reply:
OK, probably we should discard naming it "Q". Should I revert that code line?

Reply:
Yes, would be great! :)

Reply:
"Q" was removed from utils.js and useragentstrings.yml